editor/py/serve.py

```diff
@@ -90,6 +97,34 @@ def upload_dataset(file_name):
   return 'ok'
 
 
+@app.route('/dataset/<file_name>/new', methods=['POST'])
+def new_dataset(file_name):
```

Comment:
If a user wants to create a new dataset from scratch, rather than edit/modify one in place. I figured creating an empty file would be the simplest way to have this dataset appear in the `datasets.html` page that is populated based on the database directory contents

Reply:
I see. What happens if they click new, then save?

I think that would have the same effect but not require this endpoint and also allow them to change their mind.

Reply:
I could see a workflow where they create a new dataset and then have the option of saving it. However, somewhere along the way the user is going to have to specify what file name they want to use. I may leave this in as a workable if not optimal solution.